oxd-server/src/test/java/org/gluu/oxd/server/GetTokensByCodeTest.java

```diff
@@ -34,6 +37,28 @@ public void whenValidCodeIsUsed_shouldGetTokenInResponse(String host, String opH
         refreshToken(tokensResponse, client, site.getOxdId());
     }
 
+    @Parameters({"host", "opHost", "redirectUrls", "userId", "userSecret", "tokenAlgos"})
+    @Test
+    public void whenDifferentTokenAlgoUsed_shouldGetTokenInResponse(String host, String opHost, String redirectUrls, String userId, String userSecret, String tokenAlgos) {
```

Comment:
I know iterate over algorithms is easier but it's better in test to be as explicit as possible.
If we get assert failure in this test we would be forced to debug it. It wouldn't be clear for which exactly algorithm it failed.
If instead we have test list like below, then we don't need to look inside, it would be clear at glance.
```
public void getToken_withRS256() {...}
public void getToken_withRS512() {...}
public void getToken_withES256() {...}
...

```
Lets remove `tokenAlgos` and list it one by one.

Reply:
Done